contractcourt/htlc_outgoing_contest_resolver.go

```diff
@@ -18,6 +20,23 @@ type htlcOutgoingContestResolver struct {
 	htlcTimeoutResolver
 }
 
+// newOutgoingContestResolver instantiates a new outgoing contested htlc
+// resolver.
+func newOutgoingContestResolver(
```

Comment:
formatting nit (/question), do we usually wrap function params like this?

Reply:
there was an enter too many, removed.